.github/workflows/upload-source-maps-to-datadog.yml

```diff
@@ -0,0 +1,32 @@
+name: Upload Source Maps to Datadog
+
+on:
+  push:
+    branches: [boson, neutron, photon]
+
+jobs:
+  upload-source-maps:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install @datadog/datadog-ci
+        run: npm install @datadog/datadog-ci@latest
+
+      - name: Upload source maps to Datadog
+        run: |
+          export DATADOG_API_KEY=${{ secrets.DATADOG_API_KEY }}
+          ENV_NAME=boson # Where to get this?
```

Comment:
`ENV_NAME=${{ github.ref_name }}` should do the trick and give you the current branch name

Reply:
oh sweet! think I'm looking for `github.base_ref` tho right?

Reply:
oh nvm, think it's `github.event.pull_request.base.ref`, which i already got below